### PR TITLE
LPS-159267 | Audit and refactor tests for Collections.testcase

### PR DESCRIPTION
**Pull request by rafaellajordao**
Hi! :)
This pr is corresponding to this ticket: [LPS-159267](https://issues.liferay.com/browse/LPS-159267)

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

**Comment by rafaellajordao** [comment]
> ###  ci:test:tango - 174 out of 194 jobs passed in 1 hour 46 minutes
> Click here for more details.

these failures have nothing to do with the test correction :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase [hunk 1]
```diff
@@ -2190,6 +2256,11 @@ definition {
 
 		task ("Assert the correct variation is shown") {
 			AssetPublisherPortlet.viewAssetPG(assetTitle = "WC_Title_3");
```
Comment (Luizcarlosqueiroz) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase:2258]: Missing to delete this function.

**Review by Luizcarlosqueiroz — COMMENTED** [review]
Hey @rafaellajordao! It's missing only to delete one function. All the rest is good to go :)

**Comment by rafaellajordao** [comment]
ci:forward

**Pull request closed without merging**